### PR TITLE
adding navigation

### PR DESCRIPTION
**Pull request by m1k3**
refs #9 

still no styles 😛

### CODE REVIEW

File: rad-ui-ember/app/services/auth.js [hunk 1]
```diff
@@ -30,18 +30,16 @@ export default Ember.Service.extend({
...
+          this.setAuthToken(result.token);
+          this.get('apollo')
+            .query( {query: whoAmI}, 'user' )
+            .then(result => {
+              this.setUserId(result.id);
+              resolve()
+            })
+            .catch(error => reject(error));
+        })
+        .catch(error => reject(error));
```
Comment (darthjee) [inline rad-ui-ember/app/services/auth.js]: :+1:

Comment (darthjee) [reply rad-ui-ember/app/services/auth.js]: I am wondering, Since we are using graphQl, we could retrieve both id and token with a single request, right?

Comment (m1k3) [reply rad-ui-ember/app/services/auth.js]: I thought so as well, BUT the whoAmI query needs to have the `auth` header. That means that we have to:

1. Authenticate to get the token
2. Use the token in the `Authorization` header

I couldn't find a way around it so if you have suggestions I'm all 👂

Comment (darthjee) [reply rad-ui-ember/app/services/auth.js]: thinking about it, makes sense, as the first request is a mutation and the second a query

**Pull request merged**